src/components/TodoList/TodoList.tsx

```diff
@@ -1,36 +1,76 @@
-import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todoList: Todo[];
+  setSelectedTodo: () => void;
```

Comment:
The `setSelectedTodo` prop should be a function that accepts a `Todo` object, not a function that returns void. Update the type to `(todo: Todo) => void`.